connectors/spark/src/main/scala/io/unitycatalog/spark/UCSingleCatalog.scala

```diff
@@ -180,6 +180,17 @@ class UCSingleCatalog
 object UCSingleCatalog {
   val LOAD_DELTA_CATALOG = ThreadLocal.withInitial[Boolean](() => true)
   val DELTA_CATALOG_LOADED = ThreadLocal.withInitial[Boolean](() => false)
+
+  def checkUnsupportedNestedNamespace(namespace: Array[String]) = {
```

Comment:
nit: My intellij idea suggested to `Type annotation required for Unit definition `.  Maybe we can add a `Unit` for the method return type ?